tests/test_dbms.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import os
+import time
+
+import pytest
+
+import pixeltable as pxt
+from pixeltable.env import Env
+
+from .utils import reload_catalog
+
+
+class TestDbms:
+    def test_create_table(self):
+        if os.environ.get('PIXELTABLE_DB_CONNECT_STR') is None:
+            logging.error('PIXELTABLE_DB_CONNECT_STR is not set, skipping test')
+            return
```

Comment:
Use pytest.skip(); otherwise pytest will mark this test as PASSED. (There are several other tests that have internal skip conditions that you can refer to as examples)